projects/js-packages/publicize-components/src/components/form/use-connection-state.ts

```diff
@@ -33,12 +34,6 @@ export const useConnectionState = () => {
 		}
 	);
 
-	const isAutoConversionEnabled = useSelect(
-		select => select( socialStore ).isAutoConversionEnabled(),
-		[]
-	);
-	const shouldAutoConvert = isAutoConversionEnabled && isConvertible;
```

Comment:
Same here, technically it still can be turned off from Publicize RC, or if they turned off before